taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from django.conf import settings
+from django.conf.urls.static import static
+from . import views
+
+app_name = "taxi"
+
+urlpatterns = [
+    path("", views.index, name="index"),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)

```

Comment:
The static files configuration is typically handled in the project's main `urls.py` file, not in the app's `urls.py`. Consider moving this configuration to the main `urls.py` if it's not already there.